server/controllers/favorites.js

```diff
@@ -43,15 +54,18 @@ export default {
             message: 'Your list of favorite recipes is empty'
           });
         }
-        res.status(200).json(favourite);
+        res.status(200).json({
+          status: 'Success',
+          favourite
```

Comment:
The returned object name must reflect the data returned. I guess `favorites` should have served it better than `favorite`.